pkgs/top-level/all-packages.nix

```diff
@@ -21195,7 +21195,6 @@ with pkgs;
   libGLU = mesa_glu;
 
   mesa = callPackage ../development/libraries/mesa {
-    llvmPackages = llvmPackages_latest;
```

Comment:
Can you do get `llvmPackages_latest` from spliced packages or `pkgsHostHost`?

Reply:
The test `doesNotRelyOnTargetPlatform` cannot pass if we keep this override in `all-packages.nix`. But passes if we get `llvmPackages_latest` directly from arguments.

Reply:
The original override (implicitly) takes it from `pkgs`, which is indeed wrong. I am saying get it from somewhere else and then it should be fine.

Reply:
> The original override (implicitly) takes it from `pkgs`, which is indeed wrong. I am saying get it from somewhere else and then it should be fine.

Sorry I don't quite understand. Do you ask to use `llvmPackages = pkgsHostHost.llvmPackages_latest` here? I'm not a fan of leaking details (where `llvmPackages` get used in the drv) into `all-packages.nix`.